JobProcessor.App/Controllers/JobsController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using JobProcessor.App.ViewModels;
+using JobProcessor.Domain.Services;
+using System;
+using System.Web.Mvc;
+
+namespace JobProcessor.App.Controllers
+{
+    public class JobsController : Controller
+    {
+        protected readonly IJobService jobService;
+        public JobsController(IJobService jobService)
+        {
+            this.jobService = jobService;
+        }
+
+        public ActionResult Create(JobCreationViewModel job)
+        {
+            ModelState.Clear();
+            return View(job);
+        }
+
+        [HttpPost]
+        public ActionResult Create(string name, DateTime? doAfter)
+        {
+            if (ModelState.IsValid)
+            {
+                jobService.Create(name, doAfter);
+                return RedirectToAction("Success");
```

Comment:
The raw, ugly Success page is wrong. After Job is added with a success you should do one of two things: 
1. Crear data, stay on a Create Form with a small notification that Job has been added
2. Move the user to the list of jobs where he can see that Job is added

Reply:
Got it, duly noted. About that - redirection to list of jobs will be implemented along with the next feature. Current way solely in purpose to see any info.